twitter.gt2eb.style.scss

```
@@ -752,7 +752,7 @@ a.gt2-trend {
       .gt2-name {
         font-size: incr(18px);
         font-weight: 700;
-        text-decoration: underline;
+        text-decoration: none;
```

This is a good catch, but the name should still be underlined when hovering over it.
```suggestion
        &:hover {
          text-decoration: underline;
        }
```